interfaces/Portalicious/src/app/components/query-table/components/query-table-global-search/query-table-global-search.component.html

```diff
@@ -24,6 +24,25 @@
           class="absolute end-1 top-1/2 -translate-y-1/2"
         />
       }
+      @if (globalFilterValue()?.toLowerCase() === 'henry dunant') {
+        <div class="fixed bottom-0 right-0 z-50 m-4">
+          <img
+            src="assets/henry.png"
+            class="w-64"
+            alt="Gotcha!"
+            i18n-alt
+          />
+        </div>
+      } @else {
```

Comment:
Why was this added? Not sure what it does. @tijsziere @RubenGeo

Reply:
(I'm referring to the `else` block)

Reply:
@tijsziere @RubenGeo 
![image](https://github.com/user-attachments/assets/5c4e30dd-bba3-46b7-8a4e-c68e04aa0adc)


This is causing a double button in the search query component. Tbh a bit disappointing given that simply checking the feature in the browser would've shown this. I imagine that this broke after a rebase, but still, all the more reason to check things before approving/merging them. Please let's be more careful with contributions and reviews.

Reply:
(I'll open a PR to fix this).